PdCore/build.gradle

```diff
@@ -1,14 +1,17 @@
 apply plugin: 'com.android.library'
+apply plugin: 'com.github.dcendents.android-maven'
+apply plugin: 'com.jfrog.bintray'
+apply from: 'fat-aar.gradle'
+
+version = '1.0.0-rc2'
+group = 'org.puredata.android'
+archivesBaseName = 'pd-core'
 
 dependencies {
-    compile 'com.github.nettoyeurny:btmidi:9bc54ccbe1'
+    embedded project(':midi:AndroidMidi')
```

Comment:
I think this is causing "Multiple dex files define" errors when building any of the modules except PdCore.